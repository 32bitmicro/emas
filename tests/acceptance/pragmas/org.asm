.org 10
.word 1
.org 20
.word 2
