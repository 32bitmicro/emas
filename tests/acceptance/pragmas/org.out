@ 0x000a : 0x0001  /  000 000 0 000 000 001  /  1
@ 0x0014 : 0x0002  /  000 000 0 000 000 010  /  2
